### DISCUSSION TITLE
A Short Question

### DISCUSSION THREAD

**Discussion by dAtA-TRoN** [fast4x/RiMusic#1543] [Show and tell]
Which one looks better? Left or right?

![ZomboDroid_16042024084112.jpg](https://github.com/fast4x/RiMusic/assets/71184627/84f3f86a-7f22-446f-a343-734c7c21c5c3)

**Comment by OrangeZXZ** [comment]
> Which one looks better? Left or right?
> 
> ![ZomboDroid_16042024084112.jpg](https://github.com/fast4x/RiMusic/assets/71184627/84f3f86a-7f22-446f-a343-734c7c21c5c3)

The right one looks so mutch cleaner but i think the left one is the actual and is better suited for Full dark themes

Maybe make the blue less colored that way the icon in the middle blends less (the icon should be more visible)

**Reply by dAtA-TRoN** [reply]
Oh, thank you for the prompt response 🙂.

May I ask if you are somehow deeply involved in this project here?

**Reply by OrangeZXZ** [reply]
> May I ask if you are somehow deeply involved in this project here?

If I understood well yes since ViMusic death and my friend showing RiMusic ive made Issues and UI Concepts RiMusic back then was so mutch more different i hope I didn't push these changes on @fast4x too mutch 😅

**Reply by dAtA-TRoN** [reply]
> If I understood well yes since ViMusic death and my friend showing RiMusic ive made Issues and UI Concepts RiMusic back then was so mutch more different i hope I didn't push these changes on @fast4x too mutch 😅

Oh, sorry about that! There was a German instruction for ChatGPT in my response. I've removed it—it wasn't meant for you 😅. Thanks for giving me a bit of background on your role here. Is there someone responsible for finalizing graphics like the one I created here? I'm not an artist and I only manage to produce such things at a snail's pace. I wonder if anyone is interested in taking on the task. Or maybe fast4x could refine the design a bit and integrate it directly 🤔.

**Reply by OrangeZXZ** [reply]
> I wonder if anyone is interested in taking on the task. Or maybe fast4x could refine the design a bit and integrate it directly 🤔.

I always been very peeky on design and always try to find alternatives with good UI (especially on windows 11)

ViMusic and RiMusic is no exception I use them because they look cool enough and let me listen for free

I know a bit of editing but i mostly just learn C# at my school

I think fast4x is the one making the UI aspect and ikanova icons sometimes
(Sorry if there are other peoples in the background that i don't recognize)

**Reply by dAtA-TRoN** [reply]
I've also spent a looong time with ViMusic, and it was super annoying that there were never any updates. The app kept getting buggier over time, and were never getting fixed. Then I stumbled upon RiMusic, and I have to say, I'm completely blown away by fast4x. I've never seen such a drive here on GitHub, especially compared to the guy from ViMusic. Cool dude, awesome project, and lots of nice people with great ideas are around here. Can't get any better than that. It's really fun to see the app grow. But I'm digressing 😄...

Ikanakova is the icon man, I already knew that. Well, then I seem to be pretty up to speed. I think I'll just open a feature request about this thing here. fast4x will sort it out, I think. Thanks a lot for your feedback and for answering dumb questions 😁.
